support-workers/src/main/scala/com/gu/services/Services.scala

```diff
@@ -23,7 +23,7 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 
 trait ServiceProvider {
-  private lazy val config = Configuration.load()
+  private val config = Configuration.load()
```

Comment:
this is the only optimisation (so far) to make this happen during warmup.  It already makes things quicker to cold start, but I'm sure we can do more.